samples/sampler/stories/qa/Resizable.js

```diff
@@ -2,13 +2,20 @@ import Button from '@enact/sandstone/Button';
 import Item from '@enact/sandstone/Item';
 import Scroller from '@enact/sandstone/Scroller';
 import Resizable from '@enact/ui/Resizable';
+import useResizable from '@enact/ui/Resizable/useResizable';
```

Comment:
If we refactor `Resizable` to use `useResizable` inside of it, I think we don't need to modify this sampler.

Reply:
I updated `Resizable` to use `useResizable`.

Reply:
I meant if we modify `ui/Resizable` to use `useResizable`, we don't need this PR anymore. If that case, please close this PR.
:)

Reply:
Okay, I wil close this PR.